starters/gatsby-starter-blog/.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
```

Comment:
🤔 I'd like to retain this functionality. We can follow a similar approach with CircleCI here, and only run tests if these files were changed, but it just adds a bit of complexity.